api.planx.uk/modules/flows/updateTemplatedFlow/service.ts

```diff
@@ -0,0 +1,97 @@
+import type { FlowGraph } from "@opensystemslab/planx-core/types";
+import { gql } from "graphql-request";
+import merge from "lodash/merge.js";
+import { $api } from "../../../client/index.js";
+import { getFlowData } from "../../../helpers.js";
+import type { Flow } from "../../../types.js";
+
+interface GetTemplatedFlowEdits {
+  edits: {
+    data: Flow["data"]; // Not `FlowGraph` because not required to have `_root` !
+  }[];
+}
+
+interface UpdateTemplatedFlowData {
+  flow: {
+    data: FlowGraph;
+  };
+}
+
+interface InsertComment {
+  comment: {
+    id: number;
+  };
+}
+
+export const updateTemplatedFlow = async (
+  sourceFlowId: string,
+  templatedFlowId: string,
+) => {
+  const { data: sourceData, publishedFlows } = await getFlowData(sourceFlowId);
```

Comment:
`getFlowData` here is a bit tricky - if you publish your source template, then immediately edit it, by the time the `updateTemplatedFlow` event runs (staggered by 10 secs), it could _technically_ be fetching an unpublished edit or two. 

I don't think this is particularly high risk right now, but important to be clear about. This all comes back to our flattened versus unflattened data structures and not having an easy way to do hook into ShareDB via the API, which also related to initial templated flow creation. Will come back to these once initial e2e functionality !